core/src/main/resources/jenkins/security/UpdateSiteWarningsConfiguration/config.properties

```diff
@@ -23,5 +23,7 @@
 warning.core = Jenkins core: {0}
 warning.plugin = {0}: {1} 
 
-blurb = This section allows you to suppress currently applicable warnings from the update site. \
+blurb = This section allows you to suppress currently applicable warnings reported \
+  by the update site from appearing in this Jenkins instance\u2019s update notifications
```

Comment:
This is an independent administrative monitor and a sibling to the core update notification. Jenkins ships with dozens of admin monitors, so this is inaccurate.

Reply:
... from being reported for this Jenkins instance by any of its administrative monitors?

Reply:
Well, it's not _any_ of them, but the _specific_ one configured here. Perhaps something like this?

> This section allows you to suppress warnings applicable to your Jenkins configuration that are provided by the updates sites. If you do, they won't be shown by the _Update Site Warnings_ administrative monitor.